aws/resource_aws_lb_listener_rule.go

```diff
@@ -318,6 +547,27 @@ func suppressIfActionTypeNot(t string) schema.SchemaDiffSuppressFunc {
 	}
 }
 
+func suppressIfConditionFieldNotIn(fs []string) schema.SchemaDiffSuppressFunc {
```

Comment:
Even thought the name is self explained, this looks a bit complex at first sight and would definitely benefit some comments in the function about the general idea i think, what do you think?

Reply:
Added comments